### PR TITLE
Status page rework

### PR DESCRIPTION
**Pull request by maaverik**
Screenshot from 2020-10-19 00-56-19

I'm not sure where to get the token number and username from. Once those are clear, we can add that as well.

TODO: token number, username, refresh button

### CODE REVIEW

**Comment by daltonfury42** [comment]
The backend call that is made to get the aheadCount has the token info:
```
{
   "tokenId":"16619f5c-2687-4f3b-b486-afdd133f4b1b",
   "tokenNumber":1,
   "tokenStatus":"WAITING",
   "queueName":"sdadsaasdasdsa",
   "queueId":"c481fd09-4bc1-43ab-aa00-3f99e6895360",
   "aheadCount":0,
   "notifiable":true,
   "tokenCreationTimestamp":"2020-10-18T19:29:45.222+0000"
}
```

I think name went missing by mistake. Let me quickly add that.

**Comment by daltonfury42** [comment]
I've added name and contact number... if all goes well it should be deployed in a few minutes.

https://github.com/SimplQ/simplQ-backend/pull/97

File: simplq/src/components/pages/Status/QueueStatus.jsx [hunk 1]
```diff
@@ -70,7 +70,7 @@ function QueueStatus(props) {
 
   return (
     <>
-      <Header className={styles['header']}>{tokenStatusResponse.queueName}</Header>
+      <HeaderSection queueName={tokenStatusResponse.queueName} />
       <div className={styles['main-body']}>
         <TokenNumber />
```
Comment (daltonfury42) [inline simplq/src/components/pages/Status/QueueStatus.jsx:75]: Token status response has the name of the person and also the token number

**Review by daltonfury42 — APPROVED** [review]

File: simplq/src/components/pages/Status/StatusContainer.jsx [hunk 2]
```diff
@@ -15,9 +15,11 @@ export default (props) => {
...
     /* eslint-disable react/jsx-one-expression-per-line */
     status = (
-      <p>
-        People ahead of you :<span className={styles.count}>{props.aheadCount}</span>
-      </p>
+      <>
+        <p>Hello, </p>
+        <p>your courrent position is:</p>
+        <p className={styles['count']}>{props.aheadCount + 1}</p>
+      </>
```
Comment (daltonfury42) [inline simplq/src/components/pages/Status/StatusContainer.jsx:22]: can you add a 2rem margin around this?

**Comment by maaverik** [comment]
I'm getting a 500 error on trying to join. 
{"timestamp":"2020-10-18T20:01:34.811+0000","status":500,"error":"Internal Server Error","message":"No message available","path":"/v1/token"}

Any idea why?

**Comment by daltonfury42** [comment]
> I'm getting a 500 error on trying to join.
> {"timestamp":"2020-10-18T20:01:34.811+0000","status":500,"error":"Internal Server Error","message":"No message available","path":"/v1/token"}
> 
> Any idea why?

Not sure, I've tried checking out your branch locally and also on simplq.me, and am not able to reproduce the issue. Is it still happening?

**Comment by daltonfury42** [comment]
Can you take a look at mobile? 
image

Juspt putting min-width to 16rem looks like a quick fix for now

**Comment by maaverik** [comment]
> > I'm getting a 500 error on trying to join.
> > {"timestamp":"2020-10-18T20:01:34.811+0000","status":500,"error":"Internal Server Error","message":"No message available","path":"/v1/token"}
> > Any idea why?
> 
> Not sure, I've tried checking out your branch locally and also on simplq.me, and am not able to reproduce the issue. Is it still happening?

The auth fails on localhost so can't really test.
{"timestamp":"2020-10-19T05:42:08.628+0000","status":500,"error":"Internal Server Error","message":"Invalid auth token","path":"/v1/queue"}

Also, loading the homepage always gives me an error popup saying Login Failed.

**Comment by daltonfury42** [comment]
ah. are you on latest code? if not pull new-frontend

**Comment by maaverik** [comment]
> ah. are you on latest code? if not pull new-frontend

Just pulled now. Requests don't fail anymore, but the login failed popup still shows up. Also, this might be a temporary issue, but all backend requests are extremely slow for me. Takes almost 30 secs for any request. This is on the main site. Can you check on your side too?

**Comment by daltonfury42** [comment]
ah.. so this is a infra side capacity issue I need to look into.. our server stalls every two weeks... I need to set up some logging and debug this. We've been simply restarting till now.

**Pull request merged**